### PR TITLE
added some new options + lang file

### PR DESCRIPTION
**Pull request by phpust**
in this update you could specify this option inside filamentPanelProvider:
```
                FilamentSpatieLaravelHealthPlugin::make()
                    ->navigationSort(3)
                    ->navigationIcon('heroicon-o-cog-6-tooth')
                    ->navigationGroup(fn () => trans('navarak.settings'))
                    ->navigationLabel(fn () => trans('navarak.health')),
```

### CODE REVIEW

**Comment by phpust** [comment]
@shuvroroy 
change icon, menu sort, navigation group and label added+ some string added to language files.

**Review by shuvroroy — APPROVED** [review]

**Comment by shuvroroy** [comment]
@phpust can open a PR updating the readme file with adding all new addition

File: src/FilamentSpatieLaravelHealthPlugin.php [hunk 1]
```diff
@@ -42,4 +72,52 @@ public function getPage(): string
...
+    public function navigationGroup(string|\Closure|null $navigationGroup): static
+    {
+        $this->navigationGroup = $navigationGroup;
+
+        return $this;
+    }
+
+    public function getNavigationGroup(): string
+    {
+        return $this->evaluate($this->navigationGroup) ?? __('filament-spatie-health::health.navigation.group');
```
Comment (DirtyRacer1337) [inline src/FilamentSpatieLaravelHealthPlugin.php:85]: should be `filament-spatie-health::health.pages.health_check_results.navigation.group`

File: src/FilamentSpatieLaravelHealthPlugin.php [hunk 2]
```diff
@@ -42,4 +72,52 @@ public function getPage(): string
...
+    public function navigationLabel(string|\Closure|null $navigationLabel): static
+    {
+        $this->navigationLabel = $navigationLabel;
+
+        return $this;
+    }
+
+    public function getNavigationLabel(): string
+    {
+        return $this->evaluate($this->navigationLabel) ?? __('filament-spatie-health::health.navigation.label');
```
Comment (DirtyRacer1337) [inline src/FilamentSpatieLaravelHealthPlugin.php:121]: should be `filament-spatie-health::health.pages.health_check_results.navigation.labe`

**Pull request merged**